.github/workflows/pr-description-enforcer.yml

```diff
@@ -0,0 +1,14 @@
+# This workflow will enforce description on pull requests.
+name: 'PR Description Enforcer'
+on:
+    pull_request:
+        types: [opened, edited, reopened]
+
+jobs:
+    enforce:
+        runs-on: ubuntu-latest
+
```

Comment:
Isn't ubuntu-latest a bit vague? Why not a specific version?

Reply:
@n13 This is just for github runner. It is default to use ubuntu-latest, there is no need to use specific version.